### PR TITLE
Story/bods 6540

### PR DESCRIPTION
**Pull request by NitishAroraKPMG**

### CODE REVIEW

File: transit_odp/dqs/views/base.py [hunk 1]
```diff
@@ -34,7 +35,24 @@ def is_dqs_new_report(self):
...
+                org_id = self.kwargs.get("pk1")
+                if users[0].organisation_id == org_id:
+                    self.show_suppressed = True
+
+        # Call the parent class's dispatch method
+        return super().dispatch(request, *args, **kwargs)
+
     def get_queryset(self):
+
+        print("in query set")
```
Comment (pritish-ranjan01) [inline transit_odp/dqs/views/base.py:54]: Please remove

File: transit_odp/dqs/views/base.py [hunk 2]
```diff
@@ -69,6 +88,7 @@ def get_table_kwargs(self):
     def get_context_data(self, **kwargs):
 
         context = super().get_context_data(**kwargs)
+        print(f"in base: {context} ")
```
Comment (pritish-ranjan01) [inline transit_odp/dqs/views/base.py:91]: Please remove

File: transit_odp/dqs/views/suppress_observation.py [hunk 3]
```diff
@@ -16,10 +17,21 @@ def suppress_observation(self, request, **kwargs):
         """Action to suppress the observation result"""
 
         request_data = request.data
+        session_data = request.session
+        org_id = request_data.get("organisation_id", None)
+
+        if session_data and session_data.get("_auth_user_id", None):
```
Comment (pritish-ranjan01) [inline transit_odp/dqs/views/suppress_observation.py:23]: None shouldn't be defined as default for this value in this case

File: transit_odp/dqs/views/base.py [hunk 4]
```diff
@@ -34,7 +35,24 @@ def is_dqs_new_report(self):
                 self._is_dqs_new_report = False
         return self._is_dqs_new_report
 
+    def dispatch(self, request, *args, **kwargs):
+        session_data = request.session
+        self.show_suppressed = False
+        if session_data and session_data.get("_auth_user_id", None):
```
Comment (pritish-ranjan01) [inline transit_odp/dqs/views/base.py:41]: Don't need to define None default value

File: transit_odp/dqs/views/suppress_observation.py (hunk 3 above)
Comment (NitishAroraKPMG) [reply transit_odp/dqs/views/suppress_observation.py:23]: done

**Review by pritish-ranjan01 — APPROVED** [review]

**Pull request merged**